x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -66,14 +66,24 @@ const navigationTree: NavigationTreeDefinition = {
                     defaultMessage: 'Log rate analysis',
                   }),
                   link: 'ml:logRateAnalysis',
-                  icon: 'beaker',
                   getIsActive: ({ pathNameSerialized, prepend }) => {
                     return pathNameSerialized.includes(prepend('/app/ml/aiops/log_rate_analysis'));
                   },
                 },
                 {
+                  title: i18n.translate('xpack.serverlessObservability.ml.logPatternAnalysis', {
```

Comment:
@grabowskit should we have Log pattern analysis in the left hand nav for the observability project? Or is the intention for users to access this from Log Explorer / Discover?

<img width="205" alt="image" src="https://github.com/elastic/kibana/assets/7405507/32e87f18-4779-4365-99b5-1e88bb7ce599">

Reply:
Checked with @grabowskit, because pattern analysis is in Discover, please remove it from the left nav here.

Reply:
and also remove it from kibana's search bar?
![image](https://github.com/elastic/kibana/assets/22172091/d6029f21-7b30-40be-8e8c-a902e7356cd8)

Reply:
I'd keep it in the top search bar, and just hide it from the left sidenav.

Reply:
Updated in https://github.com/elastic/kibana/pull/163724/commits/6f8db8c86fd3a469915a87830e8b81f698a3772f